### PR TITLE
Drupal CDN && image work

### PR DESCRIPTION
**Pull request by LightGuard**
Multiple issues addressed here:

* DEVELOPER-1787
* DEVELOPER-1814
* DEVELOPER-1798

### CODE REVIEW

**Comment by paulrobinson** [comment]
From build error:

      Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES
section: 'excon'

**Comment by LightGuard** [comment]
Okay, looks like I merged it wrong. Delete the Gemlockfile and do bundle
install again.

On Tuesday, January 19, 2016, Paul Robinson <notifications@github.com>
wrote:

> From build error:
>
>   Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES
>
> section: 'excon'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/685#issuecomment-172793827>
> .
>


-- 
Sent from Gmail Mobile

**Comment by paulrobinson** [comment]
retest this please

**Comment by LightGuard** [comment]
Build issue doesn't appear to be my code, something happened with the checkout it seems.

**Comment by LightGuard** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by paulrobinson** [comment]
I'm trying to run this locally with:

      bundle exec ./control.rb -u --run-the-stack

I see:


Entering '_portal-quickstarts'
Entering '_sandbox-quickstarts'
Entering '_ticket-monster'
Entering '_unifiedpush-quickstarts'
Entering '_wfk-quickstarts'
running clean
Executing awestruct with args -P drupal -g --force -q
Using profile: drupal
Generating site: http://docker:32769/
Failed to initialize pipeline: No such file or directory @ rb_sysopen - _tmp/drupal8.log
rake aborted!
Error executing awestruct
/home/awestruct/developer.redhat.com/Rakefile:447:in `run_awestruct'
/home/awestruct/developer.redhat.com/Rakefile:118:in `block in <top (required)>'
Tasks: TOP => gen
(See full trace by running task with --trace)
Error running docker-compose

**Comment by paulrobinson** [comment]
OK, I guess it expects '_tmp' to be present. I've created it manually and re-running.

**Comment by LightGuard** [comment]
Hm, okay, I'll have to make sure that directory exists.

On Fri, Jan 22, 2016 at 9:53 AM, Paul Robinson <notifications@github.com>
wrote:

> OK, I guess it expects '_tmp' to be present. I've created it manually and
> re-running.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/685#issuecomment-173975751>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by paulrobinson** [comment]
Now I'm getting the error I had before deleting the _tmp dir:

Entering '_wfk-quickstarts'
running clean
Executing awestruct with args -P drupal -g --force -q
Using profile: drupal
Generating site: http://docker:32775/
Excecuting pipeline...
Unable to decrypt vault (GPGME::Error::DecryptFailed)
WARN: tilt autoloading 'tilt/sass' in a non thread-safe way; explicit require 'tilt/sass' suggested.
Generating pages...
An error occurred during output generation, all pages may not have completed during generation
Daybreak database _tmp/cache.db was not closed, state might be inconsistent
rake aborted!
Error executing awestruct
/home/awestruct/developer.redhat.com/Rakefile:447:in `run_awestruct'
/home/awestruct/developer.redhat.com/Rakefile:118:in `block in <top (required)>'
Tasks: TOP => gen
(See full trace by running task with --trace)
Error running docker-compose

**Comment by paulrobinson** [comment]
Maybe we should talk through it when you are online

**Comment by ollyjshaw** [comment]
@paulrobinson @LightGuard I've had the tmp problem too. I'm going to put a ticket in for that shortly.

**Comment by paulrobinson** [comment]
needs rebase.

**Comment by paulrobinson** [comment]
Got it working on my laptop, and saw the images being displayed.

**Comment by paulrobinson** [comment]
So just needs a rebase and a code review

File: Gemfile.lock [hunk 1]
```diff
@@ -201,16 +211,22 @@ GEM
...
     mime-types (2.1)
+<<<<<<< HEAD
+    mini_portile2 (2.1.0)
+    minitest (5.8.3)
+    minitest-reporters (1.1.7)
+=======
     mini_portile (0.6.2)
     minitest (5.8.4)
     minitest-reporters (1.1.5)
+>>>>>>> origin/master
```
Comment (paulrobinson) [inline Gemfile.lock]: merge issue

File: Gemfile.lock [hunk 2]
```diff
@@ -157,8 +162,13 @@ GEM
       multi_json (~> 1.11)
       os (~> 0.9)
       signet (~> 0.7)
+<<<<<<< HEAD
+    gpgme (2.0.12)
+      mini_portile2 (~> 2.1.0)
+=======
```
Comment (paulrobinson) [inline Gemfile.lock]: more here

File: Gemfile.lock [hunk 3]
```diff
@@ -201,16 +211,22 @@ GEM
...
       mime-types (>= 1.16, < 3)
     memoist (0.14.0)
     metaclass (0.0.4)
     method_source (0.8.2)
     mime-types (2.1)
+<<<<<<< HEAD
+    mini_portile2 (2.1.0)
+    minitest (5.8.3)
+    minitest-reporters (1.1.7)
+=======
```
Comment (paulrobinson) [inline Gemfile.lock]: And here

Comment (LightGuard) [inline Gemfile.lock]: Wow. okay, Sorry.

On Tue, Jan 26, 2016 at 11:59 AM, Paul Robinson <notifications@github.com>
wrote:

> In Gemfile.lock
> <https://github.com/redhat-developer/developers.redhat.com/pull/685#discussion_r50881785>
> :
>
> >      mail (2.6.3)
> >        mime-types (>= 1.16, < 3)
> >      memoist (0.14.0)
> >      metaclass (0.0.4)
> >      method_source (0.8.2)
> >      mime-types (2.1)
> > +<<<<<<< HEAD
> > +    mini_portile2 (2.1.0)
> > +    minitest (5.8.3)
> > +    minitest-reporters (1.1.7)
> > +=======
>
> And here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/685/files#r50881785>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp

**Pull request merged**